src/pages/SchoolPage.js

```diff
@@ -48,30 +50,76 @@ export default function SchoolPage({ location }) {
                 key={i}
                 style={{ textDecoration: "none", color: "inherit" }}
               >
-                <div
-                  style={{
-                    padding: 15,
-                  }}
-                >
-                  <span>{subjectid}</span>-<span>{subject.name}</span>
-                </div>
+                <Department>
+                  <span className="departmentCode">{subjectid}</span>
+                  <span className="departmentName">
+                    &nbsp;
+                    {subject.name}
+                  </span>
+                </Department>
               </Link>
             );
           })}
-        </div>
+        </Departments>
       )}
-    </div>
+    </PageContainer>
   );
 }
 
 SchoolPage.propTypes = {
   location: PropTypes.shape({
     search: PropTypes.string.isRequired,
+    state: PropTypes.object,
   }),
 };
 
+const deptFadeIn = keyframes`
+  from {
+    opacity: 0;
+    padding-top: 6rem;
+  }
+
+  to {
+    opacity: 1;
+    padding-top: 4rem;
+  }
+`;
+
+const PageContainer = styled.div`
+  background-color: ${grey[200]};
+  width: 100vw;
+  min-height: 100vh;
+`;
+
 const DepartmentHeader = styled.div`
   width: 100vw;
-  padding: 3vmin;
-  font-size: cacl(1vmin + 1rem);
+  padding: 2vmin 2vmin 0vmin 4vmin;
+  font-size: 2rem;
+  color: ${grey[900]};
+`;
+
+const Departments = styled.div`
+  width: 100%;
+  display: grid;
+  grid-template-columns: repeat(2, minmax(22rem, 1fr));
+  grid-gap: 0.5rem;
+  padding: 5rem 2rem 2rem 2rem;
```

Comment:
This padding-top is overwritten by the animation and seems a bit large. Maybe we can fix this by adding some padding to the top of the header or setting the final padding-top of the animation to `2rem`